Completion/X/Command/_sublimetext

```diff
@@ -7,7 +7,16 @@ _arguments -s -S : \
   '(: -)'{-h,--help}'[display help information]' \
   '(-a -n --add --new-window)'{-n,--new-window}'[open new window]' \
   '--project[load specified project]:project file:_files' \
-  '(-s --stay)'{-s,--stay}'[keep application activated]' \
+  '(--safe-mode)'--safe-mode'[Launch using a sandboxed (clean) environment]' \
```

Comment:
The other specs are in the form `'(-x --xlong-form)'{-x.--xlong-form}'[description]'` because there is a short and long form for the same setting. Since `--safe-mode` only has a long option, `'--safe-mode[Launch using a sandboxed (clean) environment]'` suffices. Same for `--launch-or-new-window`.

Reply:
I have no idea how the syntax works (and https://github.com/vapniks/zsh-completions/blob/master/zsh-completions-howto.org, https://zsh.sourceforge.io/Doc/Release/Completion-System.html#:~:text=grouping%20options don't seem to explain it adequately to my level of understanding).

Reply:
In the second link, search for "Each of the forms above may be preceded by a list in parentheses of option names and argument numbers". The grouping facility is unrelated and rarely used.

Reply:
That just advises not to reinvent the wheel when describing options. Some degree of editing is usually necessary, e.g., when the --help output of an option that takes an argument names the argument.